lib/array_intersection.rb

```diff
@@ -1,6 +1,25 @@
 # Returns a new array to that contains elements in the intersection of the two input arrays
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)
```

Comment:
Excellent use of set Elise! Can you elaborate on what does "n" represent here? Since we have two arrays.